deeppixel/image_stitching_2/code/pano.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import cv2
+import sys
+from matchers import matchers
+
+class Stitch:
+	def __init__(self, args):
+		self.path = args
+		fp = open(self.path, 'r')
+		filenames = [each.rstrip('\r\n') for each in  fp.readlines()]
+		#print filenames
+		self.images = [cv2.resize(cv2.imread(each),(480, 320)) for each in filenames]
+		self.count = len(self.images)
+		self.left_list, self.right_list, self.center_im = [], [],None
+		self.matcher_obj = matchers()
+		self.prepare_lists()
+
+	def prepare_lists(self):
+		#print "Number of images : %d"%self.count
+		self.centerIdx = self.count/2 
+		#print "Center index image : %d"%self.centerIdx"""
+		self.center_im = self.images[int(self.centerIdx)]
+		for i in range(self.count):
+			if(i<=self.centerIdx):
+				self.left_list.append(self.images[i])
+			else:
+				self.right_list.append(self.images[i])
+		#print "Image lists prepared"
+
+	def leftshift(self):
+		# self.left_list = reversed(self.left_list)
```

Comment:
Remember, we are going to document everything using sphinx, if we have docstrings for all the functions with appropriate description, then the work will be reduced and much more error free